### PR TITLE
Update german strings

### PR DESCRIPTION
**Pull request by Lineflyer**
Incorporated change from hackuhi/master, fixed typos and squashed

### CODE REVIEW

File: src/main/res/values-de/strings_pref.xml [hunk 1]
```diff
@@ -87,13 +93,22 @@
     <string name="pref_units_angle_angular_mil">Winkel</string>
     <string name="pref_units_angle_degree">Grad</string>
     <string name="pref_units_angle_desc">Wähle die Einheit für Winkelangaben</string>
+    <!-- <string name="pref_units_angle_russian_mil">Russian mil</string> -->
+    <!-- <string name="pref_units_angle_us_artillery_mil">US Artillery mil</string> -->
```
Comment (bekuno) [inline src/main/res/values-de/strings_pref.xml]: Better remove or translate?

**Comment by hakuchi** [comment]
`Russischer Strich`
`Amerikanischer Strich`

Or:
`Russischer Artillerie Strich`
`Amerikanischer Artillerie Strich`

Or:
`Russische mil`     
`Amerikanische mil`

**Comment by Lineflyer** [comment]
> I will prefer this:
Russischer Strich
Amerikanischer Strich

Never heard of those...but lets take them anyway.

**Comment by bekuno** [comment]
In this list it sounds a little bit strange.
"Strichmaß" is used in the army I remember now after reading wikipedia. 
Too late now, let it so.

**Pull request merged**